autopush/settings.py

```diff
@@ -151,6 +152,10 @@ def __init__(self,
         if 'gcm' in router_conf:
             self.routers["gcm"] = GCMRouter(self, router_conf["gcm"])
 
+
```

Comment:
Bummer. Need to remove this extra linespace and then all tests will pass. (The test is barking because of Flake8.)